cmd/clusterctl/cmd/config_provider.go

```diff
@@ -151,10 +153,15 @@ func runGetComponents() error {
 func printComponents(c client.Components, output string) error {
 	switch output {
 	case ComponentsOutputText:
+		dir, file := filepath.Split(c.URL())
+		// Remove the version suffix from the URL since we already display it
+		// separately.
+		baseURL, _ := filepath.Split(strings.Trim(dir, "/"))
```

Comment:
`Trim` will trim from both sides, is that intended? Perhaps `TrimSuffix` might be better?

Reply:
+1 to TrimSuffix (think e.g. to local repositories)